sync/src/synchronizer/block_fetcher.rs

```diff
@@ -34,6 +34,16 @@ impl BlockFetcher {
     }
 
     pub fn peer_best_known_header(&self) -> Option<HeaderView> {
+        // update best known header and clean up unknown list
+        let list = self
+            .synchronizer
+            .peers()
+            .get_unknown_list(self.peer)
+            .unwrap();
```

Comment:
Use `.expect()` instead of `.unwrap()`, when panic we can see a specific message.